### ISSUE TITLE
Accessibility test results 

### ISSUE THREAD

**Issue by KristinaGadzhieva** [FreeUKGen/FreeBMD2#617]
Nuri's accessibility test results.  Nuri prioritised the identified issues using MoSCoW prioritization technique (column A). 
https://docs.google.com/spreadsheets/d/1mTfQNm0ZE897hiR5DBqKtbfWmZyylMrw/edit?usp=sharing&ouid=101271504579904849292&rtpof=true&sd=true

To make a decision on which issues should be included in the MVP release, and separate stories will be created for those issues.

**Comment by KristinaGadzhieva** [comment]
I have reviewed the test results and added a 'Phase' column with my suggestions on which issues could be fixed in the MVP release.  @richardofsussex @DeniseColbert Could you please also have a look and leave your suggestions?
https://docs.google.com/spreadsheets/d/1mTfQNm0ZE897hiR5DBqKtbfWmZyylMrw/edit?usp=sharing&ouid=101271504579904849292&rtpof=true&sd=true

**Comment by KristinaGadzhieva** [comment]
@AlOneill Alison, could you please also have a look at the accessibility test results?

**Comment by richardofsussex** [comment]
OK, I have had a look and added my comments. Where AO has already commented I agree with her.
